src/states_screens/race_result_gui.cpp

```diff
@@ -1410,6 +1413,36 @@ void RaceResultGUI::backToLobby()
                     false, false, NULL, true /* ignoreRTL */);
             }
         }
+
+        if (race_manager->getMinorMode() != RaceManager::MINOR_MODE_SOCCER
+            && race_manager->getMinorMode() != RaceManager::MINOR_MODE_EASTER_EGG)
+        {
+            // display lap count
+            if (race_manager->getNumLaps() < 9000)
```

Comment:
Does the number `9000` hold any significance here and below `best_lap_time < 9000`, why not `9001`?